lib/roken/getaddrinfo.c

```diff
@@ -216,7 +216,7 @@ get_null (const struct addrinfo *hints,
 		       &current, const_v4, &v4_addr, NULL);
     }
     *res = first;
-    return 0;
+    return ret;
```

Comment:
'ret' must be initialized to an error that indicates "unknown family" in case hints->ai_family is not one of PF_INET6, PF_INET4, or PF_UNSPEC.